modules/ding_react/ding_react.module

```diff
@@ -356,24 +363,124 @@ function ding_react_admin_settings_form() {
     '#element_validate' => [ 'element_validate_integer_positive' ],
   ];
 
-  return system_settings_form($form);
+  $form['migration']['user_status'] = [
+    '#type' => 'fieldset',
+    '#title' => t('User migration status'),
+  ];
+
+  $form['migration']['user_status']['user_id'] = [
+    '#type' => 'textfield',
+    '#title' => t('User id'),
+    '#description' => t('The Drupal id of the user to check. Enter the id and hit TAB to search.'),
+    '#ajax' => [
+      'callback' => 'ding_react_admin_user_callback',
+      'wrapper' => 'ding-react-app-user-data',
+      'method' => 'replace',
+    ],
+  ];
+  $form['migration']['user_status']['user'] = [
+    '#prefix' => '<div id="ding-react-app-user-data">',
+    '#suffix' => '</div>',
+  ];
+  if (!empty($form_state['values']['user_id'])) {
+    $account = user_load($form_state['values']['user_id']);
+    if ($account) {
+      $form['migration']['user_status']['user']['user_account'] = [
+        '#type' => 'value',
+        '#value' => $account,
+      ];
+      $form['migration']['user_status']['user']['openlist_uid'] = [
+        '#type' => 'item',
+        '#title' => t('Openlist user id'),
+        '#markup' => (!empty($account->openlist_uid)) ? $account->openlist_uid : t('No OpenList UID for user'),
+        '#description' => t('User ids beginning with "@prefix" have already been migrated.', ['@prefix' => DING_REACT_MIGRATED_UID_PREFIX]),
+      ];
+      $form['migration']['user_status']['user']['reset'] = [
+        '#type' => 'submit',
+        '#value' => t('Reset migration status'),
+        '#submit' => ['ding_react_user_reset_migration_status'],
+      ];
+    }
+  }
+
+  $form['debug'] = [
+    '#type' => 'fieldset',
+    '#title' => t('Debugging'),
+  ];
+  $form['debug']['ding_react_debug'] = [
+    '#type' => 'checkbox',
+    '#title' => t('Enable debug logging'),
+    '#default_value' => variable_get('ding_react_debug'),
```

Comment:
`variable_get` vil default return `null` should it here have been `false`?

Reply:
In the current case it does not make a difference. The check interprets `NULL` and `FALSE` the same.